docs/src/man/reshaping_and_pivoting.md

```diff
@@ -380,3 +380,49 @@ julia> first(unstack(x, :Species, :vsum), 6)
 │ 4   │ PetalWidth  │ 0.244       │ 1.326           │ 2.026          │
 │ 5   │ id          │ 25.5        │ 75.5            │ 125.5          │
 ```
+
+To turn an `AbstractDataFrame` on its side, use [`permutedims`](@ref).
+
+```jldoctest reshape
+julia> df1 = DataFrame(a=["x", "y"], b=[1.,2.], c=[3,4], d=[true,false])
```

Comment:
apart from spaces (as we discussed earlier) - we typically write `1.0` not `1.` (here and elsewhere - for consistency)